### PR TITLE
Mobile View of Portfolio Project

### PR DESCRIPTION
**Pull request by Tamunokuro**
Included HTML and CSS files, updated the ICONS used and updated the CSS file multiple times to match the linter guidelines

### CODE REVIEW

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: INVALID CODE REVIEW :x: :stop_sign:  

Hi <STUDENT_MENTION>, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :warning:

You have submitted a project, that  does not have:
-  A descriptive title telling mewhat changes are inside.
- Also kindly note that a Pull Request should have a short summary describing introduced changes in more detail.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Make it work and then make it better!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Tamunokuro** [comment]
Hi @Nyame-Wolf please do you mean I should create a new descriptive branch and improve the summary of the pull request?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        </nav>
+    </header>
+    <section class="introduction">
+        <h1 id="intro">I'm Maraine<br>Glad to see you!</h1>
+        <h5 id="intro-details">I’m a software developer!
+            I can <br> help you build a product, 
+            feature or website <br> Look 
+            through some of my work and <br> experience! 
+            If you like what you see and <br> have a project you need coded, 
+            don’t <br> hestiate to contact me.</h5>
```
Comment (christianbiring1) [inline index.html]: - [ ] Kindly make sure this text is grabbed inside a `paragraph` tag for semantic tag's sake.

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+ul.navbar li {
+  list-style: none;
+  margin: 10px;
+}
+
+.toggler-icon {
+  color: #6070ff;
+  margin-left: 180px;
+}
```
Comment (christianbiring1) [inline css/style.css]: - [ ] Kindly make sure your page is not overflowed, make sure all the content fits on the page, no need to scroll to the left to see the hamburger menu.
Screenshot from 2022-09-22 11-25-56

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+#intro-details {
+  margin: 12px 24px 12px 24px;
+  color: #344563;
+}
+
+#intro-connect {
+  color: #7f8cff;
+  margin: 1px 24px 5px 24px;
```
Comment (christianbiring1) [inline css/style.css:67]: - [ ] Kindly make sure you stick to the origin design provided in `Figma` when it comes to `font-size` and `color` as you can see the origin design 
Screenshot from 2022-09-21 18-13-44

Comparing to your design

Screenshot from 2022-09-22 11-25-56

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @Tamunokuro , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856


## Highlights
- [ ] DEescriptive PR `Title` and `Summary`
- [ ] All linters are passing
- [ ] Flexbox is used to place the element on the web page

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    <section class="connect">
+        <h3 id="intro-connect">LET'S CONNECT</h3>
+        <ul class="socials">
+            <li><a href=""><img src="img/twitter-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/linkedin-icon.png" alt="linkedin"></a></li>
+            <li><a href=""><img src="img/medium-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/github-icon.png" alt=""></a></li>
+            <li><a href=""><img src="img/angellist-icon.png" alt=""></a></li>
+        </ul>
+    </section>
```
Comment (leonardodiasb) [inline index.html:44]: - [OPTIONAL] I suggest you wrap this Headline section in a `<section>` tag and change the 'connect' section you wrapped in a `<section>` tag to a `<div>` tag for better semantics. Please read [this article for better understanding](https://www.w3schools.com/html/html5_semantic_elements.asp).

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Tamunokuro 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - No linter errors. ✔️ 
 - UI design matching the Figma template. ✔️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**